### PR TITLE
fix(unlock-app): fixing a few issues around pricing

### PR DESCRIPTION
**Pull request by julien51**
# Description

I was having issues mostly because pricing was not fully loaded...

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/checkout/main/Confirm.tsx [hunk 1]
```diff
@@ -551,7 +558,7 @@ export function Confirm({
             <h4 className="text-xl font-bold"> {lock!.name}</h4>
             <ViewContract lockAddress={lock!.address} network={lockNetwork} />
           </div>
-          {isError && (
+          {isPricingDataError && (
             // TODO: use actual error from simulation
```
Comment (julien51) [inline unlock-app/src/components/interface/checkout/main/Confirm.tsx:562]: Maybe we can remove this line?

File: unlock-app/src/components/interface/checkout/main/Confirm.tsx [hunk 2]
```diff
@@ -452,9 +454,12 @@ export function Confirm({
       }
       case 'crypto': {
         let buttonLabel = ''
-        const isFree = pricingData!.prices.reduce((previousTotal, item) => {
-          return previousTotal && item.amount <= 0
-        }, true)
+        console.log({ isPricingDataAvailable, pricingData })
```
Comment (searchableguy) [inline unlock-app/src/components/interface/checkout/main/Confirm.tsx:457]: Remove

**Review by searchableguy — APPROVED** [review]

File: unlock-app/src/components/interface/checkout/main/Confirm.tsx (hunk 2 above)
Comment (julien51) [reply unlock-app/src/components/interface/checkout/main/Confirm.tsx:457]: oups!

Comment (julien51) [inline unlock-app/src/components/interface/checkout/main/Confirm.tsx:457]:
```suggestion
```

**Pull request merged**